testing/bucket_service.go

```diff
@@ -157,20 +159,17 @@ func CreateBucket(
 		{
 			name: "basic create bucket",
 			fields: BucketFields{
-				IDGenerator: &mock.IDGenerator{
-					IDFn: func() platform.ID {
-						return MustIDBase16(bucketTwoID)
-					},
-				},
+				IDGenerator:   mock.NewIDGenerator(bucketTwoID, t),
+				OrgBucketIDs:  mock.NewIDGenerator(bucketTwoID, t),
```

Comment:
does it matter that these are the same kind of generators?

Reply:
No, this one doesn't matter... The `IDGenerator` is used for all IDs except org/bucket.  However, I didn't remove IDGenerator here because kv would panic later if there were other changes that perhaps included creating other resources while creating buckets.